cmd/init.go

```diff
@@ -17,18 +17,14 @@ limitations under the License.
 package main
 
 import (
-	"bufio"
 	"errors"
 	"fmt"
 	"log"
 	"os"
-	"os/exec"
 	"path/filepath"
 	"strings"
 
 	"github.com/spf13/cobra"
-	flag "github.com/spf13/pflag"
-
```

Comment:
Keep this blank line to split k8s-related and external libraries in two import groups